python/modules/IcePy/Communicator.cpp

```diff
@@ -493,7 +496,7 @@ communicatorWaitForShutdown(CommunicatorObject* self, PyObject* args)
             }
             catch(const Ice::Exception& ex)
             {
-                self->shutdownException = ex.ice_clone();
+                self->shutdownException = ex.ice_clone().release();
```

Comment:
Can we have a comment for why this call to `release` is needed?

Reply:
I will add one

` self->shutdownException` is a pointer, and `ex.ice_clone()` returns an `unique_ptr`, with release we take ownership of the object.